feg/gateway/services/aaa/session_manager/client_api.go

```diff
@@ -74,3 +73,14 @@ func EndSession(in *protos.LocalEndSessionRequest) (*protos.LocalEndSessionRespo
 	}
 	return cli.EndSession(context.Background(), in)
 }
+
+func UpdateTunnelIds(in *protos.UpdateTunnelIdsRequest) (*protos.UpdateTunnelIdsResponse, error) {
+	if in == nil {
+		return nil, errors.New("Nil LocalEndSessionRequest")
+	}
+	cli, err := getSessionManagerClient()
+	if err != nil {
+		return nil, err
+	}
+	return cli.UpdateTunnelIds(context.Background(), in)
+}
```

Comment:
note `UpdateTunnelIds` may be changed to `ActivateSession` in future PRs

Reply:
haha that was my comment above. I think that would be a good change to make.